block/render.php

```diff
@@ -0,0 +1,3 @@
+<div <?php echo wp_kses_data( get_block_wrapper_attributes() ); ?>>
```

Comment:
I'm not sure this `wp_kses_data()` call is necessary here -- as it is designed to be sanitizing not escaping, (so focused on content not context), and doesn't seem to be used by the Gutenberg team when doing similar tasks:

https://github.com/WordPress/gutenberg/blob/e706d67c7482e752a4adaae3e0b0449234e19902/docs/getting-started/tutorial.md?plain=1#L685-L687

Reply:
I use it because it is there in the example for dynamic block.

https://github.com/WordPress/gutenberg-examples/blob/trunk/blocks-jsx/10-dynamic-block/template.php#L13